Octokit/Models/Request/NewAuthorization.cs

```diff
@@ -11,12 +11,35 @@ namespace Octokit
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public class NewAuthorization
     {
+        // TODO: I'd love to not need this
+        public NewAuthorization()
+        {
+        }
+
+        public NewAuthorization(string note, string[] scopes)
+        {
+            Scopes = scopes;
+            Note = note;
+        }
+
+        public NewAuthorization(string note, string[] scopes, string fingerprint)
+        {
+            Scopes = scopes;
```

Comment:
I'll cut a release branch for All The Breaking Changes before merging this, so we can think on this issue further.